### PR TITLE
Build 2.0.0b1

### PR DESCRIPTION
**Pull request by sgillies**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sgillies** [comment]
@ocefpaf is this a CI glitch or a problem with my PR? https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=625562&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=113

**Comment by sgillies** [comment]
@ocefpaf is this a CI glitch or a problem with my PR? https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=625562&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=113

**Comment by ocefpaf** [comment]
Let's try this first: @conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 channel_targets:
-  - planet_dev
-
+  - conda-forge planet_dev
```
Comment (sgillies) [inline recipe/conda_build_config.yaml:2]: :eyes:

Comment (ocefpaf) [reply recipe/conda_build_config.yaml:2]: Yeah, missed that too :-/

**Comment by sgillies** [comment]
Thanks @ocefpaf !

**Pull request merged**